src/components/dialogs/cancel.vue

```diff
@@ -74,7 +77,9 @@ export default class CancelDialog extends Mixins(PaymentMixin) {
   /** Called when user clicks "Cancel this NR" button. */
   private async confirmCancel (): Promise<void> {
     this.loading = true
-    if (await this.patchNameRequestsByAction(NrAction.CANCEL)) {
+    const data = await NamexServices.patchNameRequestsByAction(this.getNrId, NrAction.CANCEL)
+    if (data) {
+      this.setNameRequest(data)
```

Comment:
This change needs to be verified (tested).

Reply:
Ie, why is this different than before and does it work as expected now?

Reply:
all the namex-api stuff in actions were moved and as a result since they don't have access to the store the logic needed to be altered in various places. It was tested with everything else